### DISCUSSION TITLE
galaxy / play store as source?

### DISCUSSION THREAD

**Discussion by jonnyhotchkiss** [ImranR98/Obtainium#1155] [General]
I just tried to get my first app, github, via obtainium 

I then uninstalled as its early from f-droid with no updates 

Installed github,  via play store 
Installing sound switch,  via galaxy...

Like apktoid, are these addable?
Of user configurable are there instructions?

Also every time I search I have to untick gitlab source,  can the settings be saved? (fr:) thanks!

**Comment by ImranR98** [comment]
There is already an issue open for adding the Play Store as a source (#343) but it may not be doable. I've opened another issue for your search preference suggestion (#1226).

In general, feature requests and bug reports should be opened as issues instead of discussions.
